src/ts/browser/sender.ts

```diff
@@ -1,6 +1,13 @@
 import { MessageGeneric } from "../messages/types";
 import { browser } from "webextension-polyfill-ts";
 
+export const sendMessageToTab = async (
```

Comment:
Just to make sure I understand, the importance here is the use of `async` and `await`?
Seems like it's simply renaming `browser.tabs.sendMessage` otherwise.

Reply:
The objective is to hide all the low-level browser stuff from scripts and just wrap them with easy-to-use adapters. In the case of `sendMessageToTab` the adapter is quite trivial of course, could be removed.